manifests/4.5/logging.openshift.io_clusterlogforwarders_crd.yaml

```diff
@@ -0,0 +1,275 @@
+                          \n The string value may contain an integer or one of these
+                          names:     emergency alert critical error warning notice
+                          informational debug"
+                        type: string
+                      tagKey:
+                        description: TagKey specifies a record field  to  use as tag.
+                        type: string
+                      trimPrefix:
+                        description: TrimPrefix is a prefix to trim from the tag.
+                        type: string
+                    type: object
+                  type:
+                    description: Type of output plugin, for example 'syslog'
+                    type: string
+                  url:
+                    description: "URL to send log messages to. \n Must be an absolute
+                      URL, with a scheme. Valid URL schemes depend on `type`. Special
+                      schemes 'tcp', 'udp' and 'tls' are used for output types that
+                      don't define their own URL scheme.  Example: \n     { type:
+                      syslog, url: tls://syslog.example.com:1234 } \n TLS with server
+                      authentication is enabled by the URL scheme, for example 'tls'
+                      or 'https'.  See `secretRef` for TLS client authentication."
+                    type: string
+                required:
+                - name
+                - type
+                - url
+                type: object
+              type: array
+            pipelines:
+              description: Pipelines select log messages to send to outputs.
+              items:
+                properties:
+                  inputRefs:
+                    description: InputRefs lists the names of inputs to this pipeline.
+                    items:
+                      type: string
+                    type: array
+                  name:
+                    description: Name is optional, but must be unique in the `pipelines`
+                      list if provided. Required to allow patch updates to the pipelines
+                      list.
+                    type: string
+                  outputRefs:
+                    description: OutputNames lists the names of outputs from this
+                      pipeline.
+                    items:
+                      type: string
+                    type: array
+                required:
+                - inputRefs
+                - outputRefs
+                type: object
+              type: array
+          type: object
+        status:
+          properties:
+            conditions:
+              additionalProperties:
+                properties:
+                  lastTransitionTime:
+                    description: LastTransitionTime is the last time this condition
+                      changed.
+                    format: date-time
```

Comment:
do we think we should give the formatting it'll show up as? e.g. `%Y.%M.%d... etc`

Reply:
Yep, done. Its the RFC3339 format that meta.time uses, but god to mention it in the description.

Reply:
done where?